src/Apps/Artwork/Components/__stories__/ArtworkDetails.story.tsx

```diff
@@ -16,6 +19,12 @@ storiesOf("Styleguide/Artwork/", module)
             artwork={WithBothAboutConditionsAndAdditionalInfo as any}
           />
         </Section>
+        <Section title="ArtworkDetailsQueryRenderer for andy-warhol-s-and-h-green-stamps-feldman-and-schellman-11-dot-9">
+          <ArtworkDetailsQueryRenderer artwork="andy-warhol-s-and-h-green-stamps-feldman-and-schellman-11-dot-9" />
+        </Section>
+        <Section title="ArtworkDetailsQueryRenderer for ai-weiwei-arch">
+          <ArtworkDetailsQueryRenderer artwork="ai-weiwei-arch" />
```

Comment:
That is how I'm planning to use it in force or that is not how it supposed to look @damassi ?

Reply:
It should be `artworkID` to match the expected parameter sent to MP